swagger_spec_validator/common.py

```diff
@@ -30,10 +36,18 @@ def validate_json(json_document, schema_path):
     schema_path = resource_filename('swagger_spec_validator', schema_path)
     with open(schema_path) as schema_file:
         schema = json.loads(schema_file.read())
-    resolver = RefResolver('file://{0}'.format(schema_path), schema)
+    resolver = RefResolver(
+        'file://{0}'.format(schema_path),
+        schema,
+        store={'http://swagger.io/v2/schema.json': schema})
```

Comment:
Thinking about this again `api_declaration` and `resource_listing` aren't valid jsonschema, which is why they don't have `id`. They're being used as  the "json object" being validated against the swagger spec schema.

I can change this to `schema.get('id')` just to be safe.